src/Serializer/SerializerContextBuilder.php

```diff
@@ -52,12 +52,19 @@ public function createFromRequest(Request $request, bool $normalization, array $
         } elseif (isset($attributes['item_operation_name'])) {
             $operationKey = 'item_operation_name';
             $operationType = OperationType::ITEM;
+        } elseif (isset($attributes['resource_operation_name'])) {
```

Comment:
`docs_operation_name`? `metadata_operation_name`? `global_operation_name`?

Reply:
I think `resource` is clearer. In documentation, we call the corresponding attributes the resource ones (for instance: https://api-platform.com/docs/core/content-negotiation/#configuring-formats-for-a-specific-resource-or-operation => `can also be configured at resource and operation level`).
Since the attributes are directly below the `ApiResource` annotation, I think it makes sense.